contrib/localnet/orchestrator/smoketest/test_deposit_eth.go

```diff
@@ -172,13 +178,41 @@ func (sm *SmokeTest) TestDepositAndCallRefund() {
 		if err != nil {
 			panic(err)
 		}
-		if cctx.CctxStatus.Status != types.CctxStatus_Reverted || receipt.Status == 0 || *tx.To() != DeployerAddress || tx.Value().Cmp(value) != 0 {
+
+		printTxInfo := func() {
 			// debug info when test fails
 			fmt.Printf("  tx: %+v\n", tx)
 			fmt.Printf("  receipt: %+v\n", receipt)
 			fmt.Printf("cctx http://localhost:1317/zeta-chain/crosschain/cctx/%s\n", cctx.Index)
+		}
+
+		if cctx.CctxStatus.Status != types.CctxStatus_Reverted {
+			printTxInfo()
 			panic(fmt.Sprintf("expected cctx status PendingRevert; got %s", cctx.CctxStatus.Status))
 		}
+
+		if receipt.Status == 0 {
+			printTxInfo()
+			panic("expected revert tx receipt status 1; got 0")
```

Comment:
panic("expected successful status 1; got failed status 0") ?

Reply:
Yes, the message means that the tx for the revert expects a status 1
Agree it might be confusing

Reply:
Do you think we should modify the panic message or not? It's up to you as long as you think the message is resonable for test only.

Reply:
4412bd10